website/docs/configuration/segment.mdx

```diff
@@ -50,7 +50,8 @@ understand how to configure a segment.
 | `templates_logic`      | `string`     | <ul><li>`first_match`: return the first non-whitespace string and skip everything else</li><li>`join`:evaluate all templates and join all non-whitespace strings (**default**)</li></ul>                                                                                                                                   |
 | `properties`           | `[]Property` | see [Properties][properties] below                                                                                                                                                                                                                                                                                         |
 | `interactive`          | `boolean`    | when is true, the segment text is not escaped to allow the use of interactive prompt escape sequences - defaults to `false`                                                                                                                                                                                                |
-| `alias`                | `string`     | for use with [cross segment template properties][cstp]                                                                                                                                                                                                                                                                            |
+| `alias`                | `string`     | for use with [cross segment template properties][cstp]                                                                                                                                                                                                                                                                     |
+| `max_cols`             | `int`        | if the terminal width exceeds this value, the segment will be hidden. for your terminal width, see `tput cols`. Defaults to `0`
```

Comment:
`tput cols` is a unix functionality, I would replace that by adding a `oh-my-posh get cols` [command](https://github.com/JanDeDobbeleer/oh-my-posh/blob/d34d9d368c1b74b12734dfb552ab711ab7d0ade1/src/cli/get.go#L46).